src/sentry/integrations/vsts/notify_action.py

```diff
@@ -18,10 +20,11 @@ class AzureDevopsCreateTicketAction(TicketEventAction):  # type: ignore
 
     def generate_footer(self, rule_url: str) -> str:
         return "\nThis work item was automatically created by Sentry via [{}]({})".format(
-            self.rule.label,
+            # TODO(mgaeta): Bug: Rule is optional.
+            self.rule.label,  # type: ignore
             absolute_uri(rule_url),
         )
 
     @transaction_start("AzureDevopsCreateTicketAction.after")
-    def after(self, event: Event, state: str) -> Any:
-        yield super().after(event, state)
+    def after(self, event: Event, state: EventState) -> Generator[CallbackFuture, None, None]:
+        return super().after(event, state)
```

Comment:
If you return instead of yield is it still a Generator? Does the super `after` function yield?

Reply:
Not sure, so I reverted.